experimental/ag-ui/front-end/src/components/ChatAssistant.css

```
@@ -1,448 +1,539 @@
 
-.message-text blockquote {
-  border-left: 3px solid rgba(0, 0, 0, 0.2);
-  margin: 0.5rem 0;
-  padding-left: 0.75rem;
-  font-style: italic;
+.message pre,
+.message code {
+  max-width: 100%;
+  overflow: auto;
+  white-space: pre;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)